tests/system/apmserver.py

```diff
@@ -1,3 +1,8 @@
+from kibana import Kibana
+from es_helper import index_smap, index_span, index_error, apm_prefix
+from es_helper import cleanup, default_pipelines
+from helper import wait_until
+from beat.beat import INTEGRATION_TESTS, TestCase, TimeoutError
```

Comment:
I've put up a fix for this import-reordering issue in https://github.com/elastic/apm-server/pull/4459

Reply:
I reverted my changes here in favour of yours, thanks!